libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,490 @@
+:root {
+  --comparison-border-radius: 8px;
+  --comparison-border-color: #d4d4d4;
+  --comparison-column-background-color: #f8f8f8;
+  --comparison-row-border-color: #e9e9e9;
+  --comparison-desktop-max-width: 1200px;
+  --comparison-tablet-spacing: 50px;
+  --comparison-header-spacing: 24px;
+  --hover-border-color: #357BEB;
+  --primary-cell-path-color: #05834E;
+}
+
+.comparison-table .table.hide {
+  display: none;
+}
+
+.comparison-table .icon {
+  display: flex;
+  font-size: var(--type-heading-xs-size);
+}
+
+.comparison-table .icon svg {
+  top: unset;
+}
+
+.comparison-table *:is(h1, h2, h3, h4, h5, h6) {
+  line-height: 125%;
+  font-size: var(--type-heading-s-size);
+}
+
+.comparison-table .table-container {
+  max-width: calc(var(--comparison-desktop-max-width) - 2px);
+  margin: var(--spacing-xs) auto 0;
+  border: 1px solid var(--comparison-border-color);
+  border-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .header-content-dummy + .table-container {
+  margin-top: 0;
+}
+
+.comparison-table .table-container .table-column-header button {
+  width: 100%;
+  min-height: 72px;
+  border: none;
+  line-height: var(--type-heading-s-lh);
+  font-size: var(--type-heading-s-size);
+  text-align: center;
+  background-color: var(--comparison-column-background-color);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--spacing-s);
+  border-start-start-radius: var(--comparison-border-radius);
+  border-start-end-radius: var(--comparison-border-radius);
+  font-family: var(--body-font-family);
+}
+
+.comparison-table .table-container .table-column-header button:hover {
+  cursor: pointer;
+}
+
+.comparison-table .table-container .table-column-header button .toggle-icon {
+  content: url('./img/black-bg-minus.svg');
+  max-width: 24px;
+}
+
+.comparison-table .table-container .table-column-header button[aria-expanded="false"] .toggle-icon {
+  content: url('./img/plus-sign.svg');
+}
+
+.comparison-table .table-container:has(button[aria-expanded="false"]) {
+  border-end-start-radius: var(--comparison-border-radius);
+  border-end-end-radius: var(--comparison-border-radius);
+  border-start-start-radius: var(--comparison-border-radius);
+  border-start-end-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .table-container button[aria-expanded="false"] {
+  border-end-start-radius: var(--comparison-border-radius);
+  border-end-end-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .table-container .table-row {
```

Comment:
Same selector question, wouldn't `.comparison-table .table-row` suffice? I'll stop adding such comments for the rest of the file, but I would advise looking through it to see if and where the extra specificity from `.comparison-table .table-container` makes sense.